crates/iceberg-catalog/src/catalog/views/commit.rs

```diff
@@ -274,13 +283,11 @@ pub(crate) async fn commit_view<C: Catalog, A: AuthZHandler, S: SecretStore>(
         view_id,
         &parameters.view,
         metadata_location.as_str(),
-        requested_update_metadata,
+        requested_update_metadata.clone(),
         transaction.transaction(),
     )
     .await?;
-    let updated_meta = C::load_view(view_id, transaction.transaction())
-        .await?
-        .metadata;
+    // ToDo Tobi: Why did we need to load the metadata again?
```

Comment:
not sure, it may be an artifact of when we were doing partial updates on db tables instead of the load-modify-delete-create flow we're doing now